src/views/auth/Sidebar.vue

```diff
@@ -34,9 +34,12 @@ export default {
   name: 'Sidebar',
   methods: {
     singOut() {
+      // company logout
       API.companies.logout();
+      // User logout
+      localStorage.removeItem('caseId');
```

Comment:
`localStorage.removeItem('caseId');` is not related to the `Auth` -we use localStorage for Anonymous Login. Do you think we need this here?